synfig-studio/src/gui/resources/ui/vectorizer_settings.glade

```diff
@@ -61,7 +61,8 @@
   <object class="GtkDialog" id="vectorizer_settings">
     <property name="can-focus">False</property>
     <property name="title" translatable="yes">Vectorizer Settings</property>
-    <property name="default-width">340</property>
+    <property name="resizable">False</property>
+    <property name="default-width">300</property>
     <property name="default-height">220</property>
```

Comment:
I don't think they're really needed because child widgets will take care of this dialog size automatically.